xmppserver/src/main/java/org/jivesoftware/openfire/session/LocalSession.java

```diff
@@ -519,4 +524,35 @@ public String getCipherSuiteName() {
     public final Locale getLanguage() {
         return language;
     }
+
+    /**
+     * Retrieves SoftwareVersion data. This method gives access to temporary Software Version data only. You can
```

Comment:
I would add space between Software and Version. Also second sentence seems to end abruptly "You can..".